build.ps1

```diff
@@ -76,11 +101,12 @@ function Build-CMakeProject
   }
 }
 
-function Build-ToolchainX64
+function Build-Toolchain($Arch)
 {
+
   Build-CMakeProject `
     -B $BinaryCache\1 `
-    -C $SourceCache\swift\cmake\caches\Windows-x86_64.cmake `
+    -C $SourceCache\swift\cmake\caches\Windows-$Arch.LLVMName.cmake `
```

Comment:
This will try to resolve `$Arch.LLVMName.cmake`, you need `$($Arch.LLVMName).cmake`. Syntax coloring helps to catch these issues. @compnerd